src/stores/widgets/SpectralProfileWidget/SpectralProfileWidgetStore.ts

```diff
@@ -320,6 +336,37 @@ export class SpectralProfileWidgetStore extends RegionWidgetStore {
         });
     }
 
+    @computed private get intensityConfig(): IntensityConfig {
+        const frame = this.effectiveFrame;
+        if (frame) {
+            let config: IntensityConfig = {nativeIntensityUnit: frame.unit};
+            if (frame.frameInfo?.beamTable?.length > 0) {
+                const beam = frame.frameInfo.beamTable[0];
```

Comment:
If the input image has per-plane-beam table (ie there are bmaj, bmin, and bpa _per_ channel), do you consider the beam size _as a function of channel_ in the Jy/beam <-> K conversion? For line 344, do you quote the 0th elements in the beamTable as a representative bmaj and bmin for the conversion?